pkgs/applications/science/biology/neuron/default.nix

```diff
@@ -7,8 +7,9 @@
 , ncurses
 , readline
 , which
+, mpi
 , python ? null
-, mpi ? null
+, useMpi ? false
```

Comment:
```suggestion
, useMpi ? false
, mpi
```

Reply:
Could you please explain what the rationale behind the formatting changes to the inputs is?

Reply:
Packages belonging to an option belong after them in the line or in the next line ideally.